src/tools/rbd/action/Import.cc

```diff
@@ -184,6 +184,27 @@ static int do_image_snap_to(ImportDiffContext *idiffctx, std::string *tosnap)
   return 0;
 }
 
+static int get_snap_protection_status(ImportDiffContext *idiffctx, bool *is_protected)
+{
+  int r;
+  char buf[sizeof(bool)];
+  r = safe_read_exact(idiffctx->fd, buf, sizeof(buf));
+  if (r < 0) {
+    return r;
+  }
+
+  bufferlist bl;
```

Comment:
Probably a bit overkill for a single bit -- perhaps just ```*is_protected = (buf[0] != 0)```

Reply:
@dillaman  Updated, thank you.